Austin.mcdaniel/add securables endpoints

austinmcdaniel opened the pull request:
Adds support for the new EMS Securables endpoints for checking EMS permissions against specific securable items.

c-owens commented on ReleaseNotes.md:
```diff
@@ -2,4 +2,6 @@
-* Adds support for the new analytic set endpoints.
\ No newline at end of file
+* Adds support for the new analytic set endpoints.
+# Release 2.5.0
```
I failed to mention this in a previous PR but this file should only contain the newest release (so delete the previous stuff). It gets attached as the text for the github release.

austinmcdaniel replied:
ah I was wondering about that last time, I figured we just didn't have this before. Should be fixed now.

c-owens commented on src/Client/V2/Access/EmsSecurablesAccess.cs:
```diff
@@ -0,0 +1,68 @@
+        /// <param name="context">
+        /// The optional call context to include.
+        /// </param>
+        public virtual EmsSecurableEffectiveAccess GetAccessForSecurable(string securableId, string accessRight, CallContext context = null )
```
Spacing on the first argument here

austinmcdaniel replied:
fixed.

c-owens reviewed (commented):
I'm guessing something needs to be setup on d1mo to fix the test failures, but let me know if that's not the case or if there's something you want me to check out

austinmcdaniel commented:
I fixed up the test in a previous change -- I just adjusted which securable and access rights are used for the test. I think this is good to go now once this latest build succeeds.
